apps/lead/models.py

```diff
@@ -226,7 +226,10 @@ def get_for(cls, user):
         )
 
     def get_assignee(self):
-        return self.assignee.first()
+        # To be used by prefetch_related (Also, there is only one assignee)
+        assignees = self.assignee.all()
+        if assignees:
+            return assignees[0]
```

Comment:
`self.assignee.first()` will suffice

Reply:
Please read the note above

Reply:
Basically `self.assignee.first()` will not use prefetch cache.